src/Twig/ParameterExtension.php

```diff
@@ -0,0 +1,67 @@
+<?php declare(strict_types=1);
+
+namespace StefanDoorn\SyliusGtmEnhancedEcommercePlugin\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class ParameterExtension extends AbstractExtension
```

Comment:
Can we make this one `final`? In that case we promote composition over inheritence.